code/datums/keybinding/mob.dm

```diff
@@ -296,3 +296,84 @@
 		return
 	user.body_l_leg()
 	return TRUE
+
+
+
+/datum/keybinding/mob/action
+	description = "Alt click an action button and set it to the corresponding number to use this hotkey"
+	keybind_signal = "temp_lazy"
```

Comment:
don't be lazy

Reply:
oh i forgot to change them kek

Reply:
wait i did fix it, check New

Reply:
I'm pretty sure this isn't a thing already because alt clicking 1-0 as ai sets them to keybound locations that allow the ai to snap back to that spot in the future by pressing that number.